pkg/util/filehelper/file.go

```diff
@@ -152,3 +152,20 @@ func CopyDir(src string, dst string) (err error) {
 
 	return
 }
+
+// IsDirEmpty checks whether directory is empty or not
```

Comment:
I think IsEmptyDir is fine. IsEmpty is a bit ambiguous. It all comes down to what exact semantics you have in mind. Also you have to make a choice whether you treat a non existent path as _false_ (aka nope this is not a empty directory) or whether you want to treat this as error in a sense that this methods expects and existing path in which case it responds with _true_ or _false_, otherwise an error is thrown. Same for what do you do if this is actually a file and not a directory. Both approaches are valid. The other methods in this util file basically for simplicity try to avoid returning errors, but it comes down to the context in which you think we are going to use this function and whether or not it is important to differentiate.